README.md

```diff
@@ -21,19 +24,14 @@ passing user requests to downstream services - to Deskpro for 'Get help with thi
       * [Creating own customer contact forms](#creating-own-customer-contact-forms)
```

Comment:
Maybe should have section in future about accessibility? Doesn't need to be as part of this PR necessarily.